etna/generic_pages/tests/test_blocks.py

```diff
@@ -0,0 +1,54 @@
+import json
+import re
+
+from django.test import TestCase
+
+from wagtail.core.models import Site
+
+from ..models import GeneralPage
+
+
+class TestGeneral(TestCase):
+    def setUp(self):
+        root = Site.objects.get().root_page
+
+        self.general_page = GeneralPage(
+            title="General page",
+            body=json.dumps(
+                [
+                    {"type": "paragraph", "value": {"text": "This is a paragraph"}},
+                ]
+            ),
+        )
+        root.add_child(instance=self.general_page)
+
+    def assertContainsParagraph(self, subject: str, paragraph_text: str):
+        regex = re.compile(
+            "<div" + r"[\s\w'\=\-\"\.]*>\s*(.*)\s*<\/div" + ">",
+            re.MULTILINE,
+        )
+        paragraph_texts = []
+        for match in regex.finditer(subject):
+            paragraph_texts.append(match.groups()[0].strip())
+
+        self.assertIn(paragraph_text, paragraph_texts)
```

Comment:
As noted already: The paragraph block is a basic `StructBlock`, so there's no reason it wouldn't render as expected. The tests in `etna/insights/tests/test_blocks.py` are only there to test the behaviour of `SectionDepthAwareStructBlock`

Reply:
So the test is "test_paragraph_rendered_as_div" the test is testing the paragraph render in a div tag,  unless we know for sure this is the case always then maybe not needed.

Reply:
It makes sense for the tests in insights because we need to confirm that the correct heading tag is used. Whether or not the paragraph appears in a div is entirely dependent on what we put in the template, so it isn't really offering any value.

Reply:
This test is now removed